src/UPS/Service.php

```diff
@@ -176,9 +176,7 @@ public function getQuotes(QuoteRequest $request): PromiseInterface
                             ],
                         ];
                     }, $parcels),
-                    //'ShipmentRatingOptions' => [
-                    //    'NegotiatedRatesIndicator' => '',
-                    //],
+                    empty($this->credentials->getShipperNumber()) ?: 'ShipmentRatingOptions' => ['NegotiatedRatesIndicator' => '',],
```

Comment:
This is too complex for my taste. What happens to the array if the empty-invokation returns true? Do you get an empty null-value in the array?

Reply:
You are right is too complex... let me edit it